addons/hr_recruitment_survey/security/hr_recruitment_survey_security.xml

```diff
@@ -5,9 +5,19 @@
             <field name="implied_ids" eval="[Command.link(ref('survey.group_survey_user'))]"/>
             <field name="comment">It will also allow to send surveys and see the resume.</field>
         </record>
-        <record id="survey.survey_user_input_rule_survey_user_read" model="ir.rule">
-            <field name="name">Survey user input: officer: read all non private survey answers</field>
-            <field name="domain_force">[('applicant_id', '=', False)]</field>
+        <record id="survey_user_input_rule_recruitment_survey_user" model="ir.rule">
+            <field name="name">Survey user input: officer: read unrestricted survey answers or in restricted users</field>
+            <field name="model_id" ref="survey.model_survey_user_input"/>
+            <field name="domain_force">[
+                ('applicant_id', '!=', False),
+                ('survey_id.survey_type', '!=', 'appraisal'),
```

Comment:
Should we rather not have the same domain here? 
![image](https://github.com/odoo/odoo/assets/11148685/0f654dcb-e672-46b6-924b-dc0276559a14)

Now it's the same, but, still I guess it would make sense to be consistent with it.

Reply:
+1, would keep ``('survey_id.survey_type', 'in', ('assessment', 'custom', 'live_session', 'survey'))``